### PR TITLE
docs: Scala in views docs

### PR DESCRIPTION
**Pull request by patriknw**
* still rather many things left as TODO because they depend on the event sourced sample

References https://github.com/lightbend/akkaserverless-java-sdk/issues/372

### CODE REVIEW

File: docs/src/modules/java/pages/views.adoc [hunk 1]
```diff
@@ -29,24 +29,48 @@ Consider an example of a Customer Registry service with a `customer` Value Entit
...
+[.tabset]
+Java::
++
 [source,proto,indent=0]
 .src/main/proto/customer/domain/customer_domain.proto
 ----
 include::example$java-valueentity-customer-registry/src/main/proto/customer/domain/customer_domain.proto[tags=declarations;domain]
 ----
 
+[.tabset]
```
Comment (johanandren) [inline docs/src/modules/java/pages/views.adoc:41]: I think this is not right (i did the same in the value entity docs) and that it should be a single `[.tabset]` for the two language tabs rather.

**Review by johanandren — APPROVED** [review]
LGTM so far except the duplicated tabsets-tags

File: docs/src/modules/java/pages/views.adoc (hunk 1 above)
Comment (patriknw) [reply docs/src/modules/java/pages/views.adoc:41]: thanks, I'll change that

**Comment by patriknw** [comment]
I need to rebase this after https://github.com/lightbend/akkaserverless-java-sdk/pull/515 because the name change of the sample.

**Pull request merged**